src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerStepExecution.java

```diff
@@ -95,6 +95,8 @@ public void stop(@NonNull Throwable cause) throws Exception {
         closeQuietly(getContext(), decorator);
     }
 
+    @SuppressFBWarnings(value = "SE_BAD_FIELD", justification = "Warning about closeables not being non-transient non-serializable, not entirely "
+                                                                + "sure what is the desired behavior and what impact could a change have here.")
```

Comment:
This is private and only called with `ContainerExecDecorator` instance.

Could either change the class to `private static class ContainerExecCallback <T extends Closeable & Serializable>  extends BodyExecutionCallback.TailCall` and use `T` instead of `Closeable`, or directly `ContainerExecCallback` since this is actually not designed for reuse.

Reply:
Thanks a for the tip, I updated it :+1: